ddl/ddl.go

```diff
@@ -659,7 +659,6 @@ func (d *ddl) Start(ctxPool *pools.ResourcePool) error {
 	d.prepareWorkers4ConcurrencyDDL()
 	d.prepareWorkers4legacyDDL()
 
-	go d.schemaSyncer.StartCleanWork()
```

Comment:
We need to remove `metrics.DDLCounter.WithLabelValues(metrics.StartCleanWork).Inc()` in line667.